demo/packaged/validate-preamble.py

```diff
@@ -0,0 +1,46 @@
+import re
```

Comment:
Is anything calling this script? (e.g., on every build or when we create the docker images etc...)

Reply:
Will add it to Earthfile :+1:

Reply:
Addressed